docs/how_to/run_session.rst

```diff
@@ -88,7 +87,6 @@ After this time limit is reached, the session is permanently closed.
 
 Additionally, there is an *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. After a session is deactivated, a subsequent job could start an additional session.  Jobs for the new session would then take priority until the new session deactivates or is closed. After the new session becomes inactive, if the job scheduler gets a job from the original session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached.
 
-When using primitives with their context managers as previously described, the session is closed automatically when the block is exited.
```

Comment:
Can you add some words to say something like "You are encouraged to use `session.close()` to close the session when you are done submitting jobs. This allows the scheduler to run the next job without waiting for session timeout. Keep in mind, however, that you cannot submit more jobs to a closed session."